semgrep/semgrep/commands/scan.py

```diff
@@ -819,19 +815,17 @@ def scan(
     # The 'optional_stdin_target' context manager must remain before
     # 'managed_output'. Output depends on file contents so we cannot have
     # already deleted the temporary stdin file.
-    with converted_pipe_targets(target_sequence) as target_sequence:
+    with converted_pipe_targets(targets) as targets:
```

Comment:
seems iffy to shadow the variable like this

Reply:
yeah I was scared to change the logic though in case we were previously somehow depending on the shadowing :/

Reply:
fair. can fix later if ever